### DISCUSSION TITLE
NOT VERIFIED STUDENT ID

### DISCUSSION THREAD

**Discussion by Ri-shika** [community/community#29130] [GitHub Education]
I upload my school id but it didn't verify. I upload upto 2 time but it didn't verified . I follow all instruction also aa it has valid date .

**Comment by melvincwng** [comment]
@Ri-shika , Hey there, please kindly raise a ticket to [GitHub Education Support](https://support.github.com/contact/education).

Provide a few days to weeks for the support engineers to investigate your issue.

Hope this clarifies.
